### DISCUSSION TITLE
DongshanPI-ONE

### DISCUSSION THREAD

**Discussion by DongshanPI** [DongshanPI/Dongshanpi-Docs#2] [General]
東山Pi壹号开发板-在线文档讨论交流区

**Comment by lsrxh** [comment]
用git获取源码失败
``shan/manifests.git -b linux-sdk -m  SSD202D/dongshanpi-one_takoyaki_dlc00v030.xml --no-repo-verify
Get /mnt/hgfs/gongxiang/dongshanpi/repo/.git
remote: Enumerating objects: 258, done.
remote: Counting objects: 100% (258/258), done.
remote: Compressing objects: 100% (194/194), done.
remote: Total 258 (delta 62), reused 258 (delta 62)
Receiving objects: 100% (258/258), 665.44 KiB | 4.62 MiB/s, done.
Resolving deltas: 100% (62/62), done.
From /mnt/hgfs/gongxiang/dongshanpi/repo/
 * [new branch]      master     -> origin/master
fatal: No names found, cannot describe anything.
Traceback (most recent call last):
  File "/mnt/hgfs/gongxiang/dongshanpi/repo/main.py", line 628, in <module>
    _Main(sys.argv[1:])
  File "/mnt/hgfs/gongxiang/dongshanpi/repo/main.py", line 602, in _Main
    result = run()
  File "/mnt/hgfs/gongxiang/dongshanpi/repo/main.py", line 595, in <lambda>
    run = lambda: repo._Run(name, gopts, argv) or 0
  File "/mnt/hgfs/gongxiang/dongshanpi/repo/main.py", line 264, in _Run
    result = cmd.Execute(copts, cargs)
  File "/mnt/hgfs/gongxiang/dongshanpi/repo/subcmds/init.py", line 513, in Execute
    remote_ref, rev = wrapper.check_repo_rev(
AttributeError: module 'wrapper' has no attribute 'check_repo_rev'

**Reply by DongshanPI** [reply]
看着像是python版本问题。ubuntu换成python3试试。

**Comment by ChunxingJin** [comment]
老师，使用buildroot编译时，会clone linux源码，源码太大了，希望您改一下脚本。

**Comment by YuzukiTsuru** [comment]
老师，使用buildroot编译时，会clone linux源码，源码太大了，希望您改一下脚本。

**Comment by DongshanPI** [comment]
老师，使用buildroot编译时，会clone linux源码，源码太大了，希望您改一下脚本。

**Comment by Rongkangqi** [comment]
老师，使用buildroot编译时，会clone linux源码，源码太大了，希望您改一下脚本。

**Comment by YobieZhou** [comment]
老师，使用buildroot编译时，会clone linux源码，源码太大了，希望您改一下脚本。[手动狗头]

**Comment by xyx0826** [comment]
文档上的原厂烧写工具[链接](https://cowtransfer.com/s/e02e0eb2d80c42)已经过期了，请问可以更新一下吗？传到 GitHub 的话文件不会失效

文档链接：https://dongshanpi.com/DongshanPi-One/10-SupportTools/

**Comment by qinyunti** [comment]
固件烧录后找不到镜像
![image](https://github.com/user-attachments/assets/7ea33fd9-8cbd-47ef-b447-b003b0a386ce)
